client/src/Components/QuoteButton/QuoteButton.jsx

```diff
@@ -1,12 +1,21 @@
 import React from 'react';
 import {Button} from '@material-ui/core';
+// import { useState } from 'react';
 
 export default function QuoteButton() {
-    return (
-        <div>
-            <Button variant="contained" color="secondary">
-                Genie, Inspire Me
-            </Button>
-        </div>
-    )
+  // const [quote, setQuote] = useState('');
+
+  // Click handler method
+  const handleButtonClick = function(){
+    console.log('The button was clicked!');
```

Comment:
Create Quote component to be triggered when button is clicked in place of console.log.